lightning/src/onion_message/messenger.rs

```diff
@@ -396,7 +399,7 @@ where
 			},
 		};
 
-		log_trace!(self.logger, "Responding to onion message with path_id {:02x?}", path_id);
+		log_trace!(self.logger , "Responding to onion message {} with path_id {:02x?}", response , path_id);
 
 		if let Err(e) = self.send_onion_message(path, response, None) {
```

Comment:
You'll want to update the log in `handle_onion_message` as well to log the message contents.